CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # CHANGELOG
 
 
+## v0.6.4
+* Adjusting support to move cursor to the end of last text block.
```

Comment:
@bhsiao55 i believe this change is associated with https://github.com/wb-apps/tmz-cms/issues/922

Reply:
@efjacobson I changed to match the title from https://github.com/wb-apps/tmz-cms/issues/922